pkg/deploy/util/util.go

```diff
@@ -37,6 +37,11 @@ func LabelForDeploymentConfig(config *deployapi.DeploymentConfig) string {
 	return fmt.Sprintf("%s/%s:%d", config.Namespace, config.Name, config.LatestVersion)
 }
 
+// VersionedLabelForDeploymentConfig builds a string identifier for a given version of the DeploymentConfig.
+func VersionedLabelForDeploymentConfig(config *deployapi.DeploymentConfig, version int) string {
+	return fmt.Sprintf("%s/%s:%d", config.Namespace, config.Name, version)
```

Comment:
I wasn't sure about it either (helped in printing the names of older deployments). 

removed